python/pyspark/sql/connect/expressions.py

```diff
@@ -420,6 +420,12 @@ def __repr__(self) -> str:
             return f"{self._name}({', '.join([str(arg) for arg in self._args])})"
 
 
+class UnresolvedBinaryFunction(UnresolvedFunction):
```

Comment:
We could just remove this `UnresolvedBinaryFunction` and fix the doctests like:

```
    >>> df.age + 1
    Column<...>
    >>> 1 / df.age
    Column<...>
```

for now